pkg/controller/nodesetup/sync_raids.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2023 ScyllaDB.
+
+package nodesetup
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	scyllav1alpha1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1alpha1"
+	"github.com/scylladb/scylla-operator/pkg/disks"
+	"github.com/scylladb/scylla-operator/pkg/util/blkutils"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/klog/v2"
+)
+
+func (nsc *Controller) syncRaids(ctx context.Context, nc *scyllav1alpha1.NodeConfig) ([]metav1.Condition, error) {
+	var errs []error
+	var progressingConditions []metav1.Condition
+
+	if nc.Spec.LocalDiskSetup == nil {
+		return progressingConditions, nil
+	}
+
+	blockDevices, err := blkutils.ListBlockDevices(ctx, nsc.executor)
+	if err != nil {
+		return progressingConditions, fmt.Errorf("can't list block devices: %w", err)
+	}
+
+	udevControlEnabled := false
+	if _, err := os.Stat("/run/udev/control"); !os.IsNotExist(err) {
```

Comment:
so i this fails with different error then IsNotExists then `udevControlEnabled == true`?